compiler/one-cmds/onecc

```diff
@@ -114,6 +115,40 @@ def _parse_arg(parser):
     return args
 
 
+def _verify_codegen(parser, args):
+    """
+    verify one-codegen arguments
+
+    This verification logic comes from 'one-codegen' codes.
+    """
+    cfg_args = SimpleNamespace()
+    oneutils.parse_cfg(args.config, 'one-codegen', cfg_args)
+    cmd_backend_exist = oneutils.is_valid_attr(args, 'backend')
+    cfg_backend_exist = oneutils.is_valid_attr(cfg_args, 'backend')
+    cfg_backends_exist = oneutils.is_valid_attr(cfg_args, 'backends')
+
+    if cfg_backend_exist and cfg_backends_exist:
+        parser.error(
+            '\'backend\' option and \'backends\' option cannot be used simultaneously.')
```

Comment:
(optional) if syntax coloring in other tools are also incorrect, how about using like this?
```py
parser.error("'backend' option and 'backends' option cannot be used simultaneously.")
```